ecosystem/sep-0038.md

```diff
@@ -0,0 +1,176 @@
+
+`delivery_methods` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`name` | string | The value to use when making `POST /quote` requests.
+`description` | string | A human readable description of the method identified by `name`.
+
+### GET Prices
+
+This endpoint can be used to fetch the [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) prices of available off-chain assets in exchange for a Stellar asset and vice versa.
+
+#### Request
+
+Only one of the following parameters should be included in the request.
+
+Name | Type | Description
+-----|------|------------
+`asset` | string | (optional) The [SEP-32](sep-0032.md) address of the Stellar asset you want to get the price of available off-chain currencies in. Lumens should be specified as `XLM`.
+`currency` | string | (optional) The [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) code of the currency you want to get the price of available Stellar assets in.
+
+#### Response
+
+Only one of the following attributes should be included in the response.
+
+Name | Type | Description
+-----|------|------------
+`assets` | array | (optional) An array of objects containing the available Stellar assets and their prices in terms of the `currency` provided in the request.
+`currencies` | array | (optional) An array of objects containing the available currencies and their prices in terms of the `asset` provided in the request.
+
+`assets` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`asset` | string | Same as the definition of `asset` in the request.
+`price` | string | The price of one unit of `asset` in terms the `currency` sent in the request.
+`decimals` | integer | The number of decimals needed to represent asset.
+
+`currencies` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`currency` | string | Same as the definition of `currency` in the request.
+`price` | string | The price of one unit of `currency` in terms of the `asset` sent in the request.
+`decimals` | integer | The number of decimals needed to represent the currency.
+
+### GET Price
+
+This endpoint can be used to fetch the indicative price of a Stellar asset and currency [pair](https://en.wikipedia.org/wiki/Currency_pair).
+
+#### Request
+
+Name | Type | Description
+-----|------|------------
+`base` | string | The base asset or currency. Ex. `USDC:circle.io`, `ARS`
+`counter` | string | The counter asset or currency.
+`base_amount` | string | The amount of the base asset or currency to be traded for the counter asset or currency.
+
+#### Response
+
+Name | Type | Description
+-----|------|------------
+`price` | string | The price of one unit of the base asset or currency in terms of the counter asset or currency for the `base_amount` specified in the request.
```

Comment:
💡  What's the motivation for the term `price`? When I worked on Vibrant's internal API for quotes I remember seeing that exchange rate APIs often referred to the resulting value as the `rate` and we used the same term there. If `rate` is a more common term than `price` it may be easier for readers to understand.

Reply:
The inspiration  was the [0x API](https://0x.org/docs/api#request-1), but I'm not married to the term.

Reply:
I don't know what is better. Stick with price. I'll resolve this one.